packages/core/parcel/src/cli.js

```diff
@@ -103,6 +104,30 @@ const commonOptions = {
     },
     [],
   ],
+  '-F, --feature-flag <name=value>': [
```

Comment:
Nitpick: Honestly prefer without the `-F` shorthand. People aren't going to type this by hand anyway so explicit is better.

Reply:
Also can you set multiple flags?

Reply:
I think it might need to be `--feature-flag <name=value...>` as per [this](https://github.com/tj/commander.js/blob/master/examples/options-variadic.js)?

But the input would be like `parcel build --feature-flag foo=true bar=false` whereas I normally prefer `parcel build --feature-flag foo=true --feature-flag bar=false` but not sure how easy that would be

Reply:
The latter is what works out the box with `commander` (i.e. `parcel build --feature-flag foo=true --feature-flag bar=false`), that was my intention rather than being able to set multiple flags with one option (I would expect most of time time it'll be some sort of launcher script configuring these anyway, and if you're setting a flag by hand it's usually a single one.

Reply:
Perfect 👍